nrt/nrt.py

```diff
@@ -531,6 +534,24 @@ def main():
 
         log_message += " pattern filter"
 
+    directory_failed = []
+    if (options.failed == True):
+        for root, dirs, files in os.walk(output_dir):
+            for file in files:
+                if "info.txt" == file:
+                    test_case = nrtDiff.TestCase("dummy")
```

Comment:
what "dummy" stays for?

Reply:
the parsing works only if a testcase name was defined, so we provide a dummy one

Reply:
ok, thx